bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2015A-kr.xml

```diff
@@ -1,5 +1,5 @@
 <conf>
-    <smtp>smtp.cl.gemini.edu</smtp>
+    <smtp>sbfsmtprelay-lv1.cl.gemini.edu</smtp>
 
```

Comment:
I think it is for good, it will relay emails through gmail instead of the old exchange protocol